extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -94,7 +96,8 @@ export class AmpAnim extends AMP.BaseElement {
   /** @override */
   unlayoutCallback() {
     // Release memory held by the image - animations are typically large.
-    this.img_.src = '';
+    this.img_.src = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///' +
+        'yH5BAEAAAAALAAAAAABAAEAAAIBRAA7';
```

Comment:
Not the [tiniest gif ever](http://probablyprogramming.com/2009/03/15/the-tiniest-gif-ever)?